invenio_records_resources/services/records/components.py

```diff
@@ -133,3 +135,20 @@ class RelationsComponent(ServiceComponent):
     def read(self, identity, record=None):
         """Read record handler."""
         record.relations.dereference()
+
+
+class BackRelationsComponent(ServiceComponent):
+    """Back Relations service component."""
+
+    def update(self, identity, data=None, record=None, uow=None, **kwargs):
+        """Register a task for the update propagation."""
+        # FIXME: we need a better way... service.name would be ok
+        # or registry.service_id(self.service)
+        record_type = str(record.__class__.__name__).lower() + "s"
+        # FIXME: until the run_components has been fixed the uow
+        # is passed as a cmp attr instead of param.
+        self.uow.register(TaskOp(
```

Comment:
So records that wants to use this change propagation, will simply add the this component in their config?

Reply:
yes, along with the `indexed_at` (dumper ext + cmp)